src/main/java/frc/robot/subsystems/Vision/IOLimeLight.java

```diff
@@ -0,0 +1,68 @@
+package frc.robot.subsystems.Vision;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.net.PortForwarder;
+import edu.wpi.first.networktables.*;
+
+public class IOLimeLight implements VisionIO {
+
+    private final AprilTagFieldLayout fieldLayout;
+    private final NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+
+    public IOLimeLight(int AprilID) {
+        PortForwarder.add(5800, "10.59.87." + AprilID, 5800);
+        try {
+            fieldLayout = AprilTagFieldLayout.loadFromResource(AprilTagFields.k2023ChargedUp.m_resourceFile);
+        } catch (Throwable t) {
+            throw new RuntimeException(t);
+        }
+
+
+    }
+
+
+    @Override
+    public void setPipeLine(int pipeLineIndex) {
+        table.getEntry("pipeline").setNumber(pipeLineIndex);
+    }
+
+    @Override
+    public void updateInputs(VisionInputs inputs) {
+        inputs.yaw = table.getDoubleTopic("tx").getEntry(0.0).get();
+
+        inputs.pitch = table.getDoubleTopic("ty").getEntry(0.0).get();
+
+        inputs.area = table.getDoubleTopic("ta").getEntry(0.0).get();
+
+        inputs.latency = table.getDoubleTopic("tl").subscribe(0.0).get(); //Todo: add latency to IOPhoton
```

Comment:
Why do you subscribe here and don't `getEntry` like the other fields?

Reply:
Actually, I have no Idea but I changed it and hope that the robot doesn't go kablooew.